plugins/Events/Events.php

```diff
@@ -143,7 +143,8 @@ public function configureViewDataTable(ViewDataTable $view)
         $secondaryDimension = $this->getSecondaryDimensionFromRequest();
         $view->config->subtable_controller_action = API::getInstance()->getActionToLoadSubtables($apiMethod, $secondaryDimension);
 
-        if (Common::getRequestVar('pivotBy', false) === false) {
+        $pivotBy = Common::getRequestVar('pivotBy', false);
+        if (empty($pivotBy)) {
```

Comment:
When unsetting the pivot `pivotBy`is set to the string `0`. This wasn't catched here, so the displayed columns were incorrect in this case.